node/build_node_bridge.sh

```diff
@@ -18,6 +18,8 @@ Usage: $(basename "$0") [-d] [-o DIR/]
 Options:
 	-d -- debug build (default is release, follows \$CONFIGURATION_NAME)
 	-o -- where to copy the built module (default: build/\$CONFIGURATION_NAME)
+
+NODE_OS_NAME must also be set to the value of Node's \`os.platform()\`.
```

Comment:
Could be worth mentioning that this is an environment variable for a newbie, but maybe not worth it.

Reply:
I think in practice it'll be rare to invoke this explicitly at all, but maybe *that* note should be there too.